scripts/test_upcoming_pair_quote.py

```
@@ -0,0 +1,114 @@
+
+    print(f"\n[3/5] Placing 2 REAL limit orders on Polymarket CLOB:")
+    print(f"      - Order 1: BUY UP   @ ${args.price:.2f} ({args.size} shares)")
+    print(f"      - Order 2: BUY DOWN @ ${args.price:.2f} ({args.size} shares)")
+
+    res_up = engine.place_live_quote(token_id=up_token, price=args.price, size=args.size, side="BUY")
+    res_dn = engine.place_live_quote(token_id=down_token, price=args.price, size=args.size, side="BUY")
+
+    up_id = res_up.get("order_id") if res_up else None
+    dn_id = res_dn.get("order_id") if res_dn else None
+
+    print(f"\n[OK] ORDERS PLACED SUCCESSFULLY!")
```

Ruff (0.16.3)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)